frontend/readline.c

```diff
@@ -8,6 +8,12 @@
 #include <string.h>
 #include <signal.h>
 #include <unistd.h>
+
+#undef __generic
```

Comment:
Why do we need this?

Reply:
To get the correct functions on FreeBSD:
https://github.com/freebsd/freebsd/blob/release/12.1.0/include/libgen.h#L44

There may be a better way, and maybe this is fixed/changed in upstream FreeBSD anyways.

Reply:
I wasn't able to find a better way to achieve this.